### PR TITLE
Feature/31 create a room in ddb

### LINKED ISSUE #31: As a user, when creating a new room, my join room request is processed appropriately on AWS

### PR DESCRIPTION
**Pull request by Startouf**
- [x] Based on pr #33 and recent merges from master

🥵
Screenshot 2020-05-08 at 22 16 51

Resolves #31

### CODE REVIEW

File: extension/src/background/websocket-client.ts [hunk 1]
```diff
@@ -20,10 +20,46 @@ export default class WebSocketClient {
                 );
                 resolve();
             };
+            this.webSocket.onmessage = (event: any) => {
```
Comment (MollardMichael) [inline extension/src/background/websocket-client.ts:23]: The type is string here no?

File: extension/src/types.ts [hunk 2]
```diff
@@ -81,15 +81,15 @@ export class Extension {
     }
 }
 
-export enum MessageType {
+export enum MessageType { // those values must be in sync with AWS lambda routes of servrless.yml
     DEBUG_MESSAGE,
-    CHANGE_ROOM,
+    CHANGE_ROOM = "join-room",
```
Comment (MollardMichael) [inline extension/src/types.ts:86]: We should teenage the even value too.

Comment (Startouf) [reply extension/src/types.ts:86]: teenage ?

**Comment by Startouf** [comment]
@mourtisma 

> Aussi je veux bien vos avis sur ces erreurs de typescript-eslint qui me paraissent peut-être un peu despotiques <https://github.com/LesGars/WatchWithMe/pull/86/checks?check_run_id=659373915>

Aussi, je n’arrive pas à voir ces erreurs sur mon VSCode, vous y arrivez vous ?
<sub>[View in Slack](https://lesgarshack.slack.com/archives/CUHTQKV9N/p1589047528001800)</sub>

File: extension/src/types.ts (hunk 2 above)
Comment (MollardMichael) [reply extension/src/types.ts:86]: Sorry. My phone autocorrect changed my sentence 

I wanted to say that we should also use JOIN_ROOM as the event value

File: extension/src/background/websocket-client.ts (hunk 1 above)
Comment (Startouf) [reply extension/src/background/websocket-client.ts:23]: Does not seem to be so simple https://stackoverflow.com/questions/9462879/how-do-i-tell-the-type-of-websocket-onmessages-parameter

File: extension/src/types.ts (hunk 2 above)
Comment (Startouf) [reply extension/src/types.ts:86]: ok, but is it fine to have API Gateway routes/urls with uppercase (cf serverless.yml) ?

**Pull request merged**